tutorials/bios-boot-tutorial/README.md

```diff
@@ -16,12 +16,12 @@ The `bios-boot-tutorial.py` script simulates the bios boot sequence.
 
 2. Install the latest [CDT binaries](https://github.com/AntelopeIO/cdt/releases) by following the steps provided in the README.
 
-3. Compile the latest [EOS System Contracts](https://github.com/eosnetworkfoundation/eos-system-contracts/releases). Replaces `release/*latest*` with the latest release branch.
+3. Compile the latest [System Contracts](https://github.com/eosnetworkfoundation/system-contracts/releases). Replaces `release/*latest*` with the latest release branch.
 
 ```bash
 $ cd ~
-$ git clone https://github.com/eosnetworkfoundation/eos-system-contracts
-$ cd ./eos-system-contracts/
+$ git clone https://github.com/VaultaFoundation/system-contracts
+$ cd ./system-contracts/
```

Comment:
Seems we should load the `core.vaulta` contract.

Also might as well address this while your at it:
https://github.com/VaultaFoundation/system-contracts/issues/183#issuecomment-2887591053

Although, maybe those changes should wait until `cleos` is updated.

Reply:
These are the additional accounts I have found are needed. Some are new with the tokenomics and ram updates. 
```
eosio.fees 
eosio.reward 
eosio.wram 
eosio.reserv 
```

We could consider adding 
```
vaulta 
core.vaulta 
```

We don't need to wait for cleos to be updated, push action works fine.

Reply:
Updates to resolve
- Added the missing accounts
- Added `A` token support with vaulta contracts and permissions
- Added pytest to validate 
- Backwards compatibility, when no `vaulta-contracts-dir` arg is supply reverts to previous behavior